CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Changelog
 ================================================================================
 
+2210-2
+--------------------------------------------------------------------------------
+
+### General Notes
+
+⚠️ Before upgrading to this version make sure that there no workflows
+running on older versions of the integration anymore. The new Settings are not
+compatible with older workflow processes.
+
+### Main Changes
+
+* Documentation now clearly states that it is recommended to store the `apiKey`
+in the properties file.
+
+* Configuration has been refactored so that the GlobalLink settings are not 
+automatically published with the pages that they are linked to. 
+
+  **Upgrade Steps:**
+  1. Create new folders for your global- and site-specific Settings at  
+  `/Settings/Options/Settings/Translation Services` and
+  `<SITE_ROOT>/Options/Settings/Translation Service`
+  2. Move the existing GlobalLink settings into the global- and site-specific 
```

Comment:
State naming convention for settings document explicitly? In apps/studio-client/shared/js/gcc/src/GccWorkflowPlugin.ts it looks like the settings document's name is fixed.

Are these site-specific folders required in any case or just, if site-specific GlobalLink settings are used? Is the global settings folder required in any case as soon as the extension is active?